src/rgw/rgw_sal_motr.cc

```diff
@@ -1480,26 +1480,10 @@ int MotrObject::get_obj_state(const DoutPrefixProvider* dpp, RGWObjectCtx* rctx,
   *_state = state;
 
   // Get object's metadata (those stored in rgw_bucket_dir_entry).
-  bufferlist bl;
```

Comment:
@SeagateChaDeepak : Why is this cache code removed?

Reply:
As get_bucket_dir_ent() function handles all the version related changes, this should be fine.